### DISCUSSION TITLE
Does anyone knows how to fix this, i have been trying all day but nothing is working

### DISCUSSION THREAD

**Discussion by MedX101** [shuding/nextra#2503] [General]
Processing css\styles.css...
│ Error: Loading PostCSS Plugin failed: Cannot find module '../lightningcss.win32-ia32-msvc.node'
│ Require stack:
│ - C:\Users\MedX\infosnapz\node_modules\.pnpm\lightningcss@1.22.0\node_modules\lightningcss\node\index.js
│ - C:\Users\MedX\infosnapz\node_modules\.pnpm\postcss-lightningcss@1.0.0_postcss@8.4.31\node_modules\postcss-lightningcss\src\i…
│ - C:\Users\MedX\infosnapz\packages\nextra-theme-docs\postcss.config.js
│ (@C:\Users\MedX\infosnapz\packages\nextra-theme-docs\postcss.config.js)
│     at load (C:\Users\MedX\infosnapz\node_modules\.pnpm\postcss-load-config@4.0.1_postcss@8.4.31\node_modules\postcss-load-con…
│     at C:\Users\MedX\infosnapz\node_modules\.pnpm\postcss-load-config@4.0.1_postcss@8.4.31\node_modules\postcss-load-config\sr…
│     at Array.map (<anonymous>)
│     at plugins (C:\Users\MedX\infosnapz\node_modules\.pnpm\postcss-load-config@4.0.1_postcss@8.4.31\node_modules\postcss-load-…
│     at processResult (C:\Users\MedX\infosnapz\node_modules\.pnpm\postcss-load-config@4.0.1_postcss@8.4.31\node_modules\postcss…
│     at C:\Users\MedX\infosnapz\node_modules\.pnpm\postcss-load-config@4.0.1_postcss@8.4.31\node_modules\postcss-load-config\sr…
│     at async Promise.all (index 0)
│  ELIFECYCLE  Command failed with exit code 1.
│  ELIFECYCLE  Command failed with exit code 1.
└─ Failed in 49.8s at C:\Users\MedX\infosnapz\packages\nextra-theme-docs
 ELIFECYCLE  Command failed with exit code 1.
PS C:\Users\MedX\infosnapz>

**Comment by dimaMachina** [comment]
provide reproduction, and open an issue not discussion
